### PR TITLE
Support oracle udf remainder in presto

### PR DESCRIPTION
**Pull request by shenh062326**
Test plan - (Please fill in how you tested your changes)

Please make sure your submission complies with our [Development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [Formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), and [Commit Message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests) guidelines. Don't forget to follow our [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution) for any code copied from other projects.

Fill in the release notes towards the bottom of the PR description.
See [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) for details.

```
== RELEASE NOTES ==

General Changes
* Add udf remainder int MathFunctions
```

### CODE REVIEW

**Review by rongrong — CHANGES_REQUESTED** [review]
Thanks for contributing. Please add unit tests in `TestMathFunctions` and documentation in `math.rst`. Thanks!

**Comment by shenh062326** [comment]
> Thanks for contributing. Please add unit tests in `TestMathFunctions` and documentation in `math.rst`. Thanks!

Done.

File: presto-docs/src/main/sphinx/functions/math.rst [hunk 1]
```diff
@@ -133,6 +133,10 @@ Mathematical Functions
 
     Returns the modulus (remainder) of ``n`` divided by ``m``.
 
+.. function:: remainder(n, m) -> [same as input]
+
+    Like oracle remainder, it returns the modulus (remainder) of m - (n * Math.round(m / n)).
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/functions/math.rst:138]:
```suggestion
    Returns the modulus (remainder) of m - (n * Math.round(m / n)).
```

File: presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java [hunk 2]
```diff
@@ -563,6 +563,86 @@ public void testMod()
...
+                assertFunction("remainder(DOUBLE '" + left + "', DOUBLE '" + right + "')", DOUBLE, (right - (left * Math.round(right / left))));
+            }
+        }
+
+        for (double left : doubleLefts) {
+            if (left == 0) {
+                continue;
+            }
+            for (double right : doubleRights) {
+                float fl = (float) left;
```
Comment (tdcmeehan) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java:639]: Call these floatLeft and floatRight for consistency.

**Pull request closed without merging**